packages/components/src/Stepper/Stepper.component.test.js

```diff
@@ -50,5 +50,28 @@ describe('Stepper Component', () => {
 			expect(renderActions).toHaveBeenCalledWith(true);
 			expect(wrapper.getElement()).toMatchSnapshot();
 		});
+
+		it('should render when there is an aborted in the steps', () => {
+			// given
+			const title = 'Test';
+			const steps = [
+				{
+					label: 'Fetch Sample',
+					status: LOADING_STEP_STATUSES.SUCCESS,
+				},
+				{ label: 'Flattening', status: LOADING_STEP_STATUSES.ABORTED },
```

Comment:
Does this mean that if I abort the flattening step, I need to auto abort the next too?
Can we simplify it by saying if only one is aborted?